.github/workflows/ci-arch.yml

```diff
@@ -53,11 +53,11 @@ jobs:
         popd
     - name: Download archlinux-keyring
       run: |
-        wget "https://sources.archlinux.org/other/archlinux-keyring/archlinux-keyring-20200108.tar.gz"
-        tar -xzf archlinux-keyring-20200108.tar.gz
+        wget "https://sources.archlinux.org/other/archlinux-keyring/archlinux-keyring-20200622.tar.gz"
+        tar -xzf archlinux-keyring-20200622.tar.gz
```

Comment:
sad we need to update the keyring like this. Is there a -latest or a git repo we could download it from?